src/ducks/aggregations.js

```diff
@@ -0,0 +1,49 @@
+import { request, GraphQLClient } from 'graphql-request';
+
+const initialState = {
+ data: null,
+ selectedPreviousAnnotation: null
+};
+
+const TEMP_WORKFLOW_ID = '3017';
+const TEMP_SUBJECT_ID = '72815';
+const CAESAR_HOST = 'https://caesar-staging.zooniverse.org/graphql';
```

Comment:
These temporary constants are functional for testing, but we may need to start setting up our dynamic config file with these values [(example config.js)](https://github.com/zooniverse/edu-api-front-end/blob/master/src/lib/config.js) soon before we forget.